specification/sql/resource-manager/Microsoft.Sql/preview/2022-05-01-preview/examples/ManagedDatabaseMoveOperationResultGet.json

```diff
@@ -0,0 +1,49 @@
+{
+  "parameters": {
+    "subscriptionId": "00000000-1111-2222-3333-444444444444",
+    "resourceGroupName": "rg1",
+    "locations": "westeurope",
+    "operationId": "15961324-d809-46ed-86b9-d786953140e2",
+    "api-version": "2022-05-01-preview"
+  },
+  "responses": {
+    "200": {
+      "properties": {
```

Comment:
Does this format is expected? Without body tag?

Reply:
Fixed in #20066